### PR TITLE
stmtsummary: add system vars description

### PR DESCRIPTION
**Pull request by shichun-0415**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
This is an addition to https://github.com/pingcap/docs/pull/12069

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/13032
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: system-variables.md [hunk 1]
```diff
@@ -3797,6 +3797,63 @@ For details, see [Identify Slow Queries](/identify-slow-queries.md).
...
 
+### tidb_stmt_summary_enable_persistent <span class="version-mark">New in v6.6.0</span>
+
+> **Warning:**
+>
+> Statements summary persistence is an experimental feature. It is not recommended that you use it in the production environment. This feature might be changed or removed without prior notice. If you find a bug, you can report an [issue](https://github.com/pingcap/tidb/issues) on GitHub.
+
+- Scope: GLOBAL
+- Type: Boolean
+- Default value: `false`
```
Comment (Oreoxmt) [inline system-variables.md:3808]:
```suggestion
- Default value: `OFF`
```

File: system-variables.md [hunk 2]
```diff
@@ -3797,6 +3797,63 @@ For details, see [Identify Slow Queries](/identify-slow-queries.md).
...
+
+### tidb_stmt_summary_filename <span class="version-mark">New in v6.6.0</span>
+
+> **Warning:**
+>
+> Statements summary persistence is an experimental feature. It is not recommended that you use it in the production environment. This feature might be changed or removed without prior notice. If you find a bug, you can report an [issue](https://github.com/pingcap/tidb/issues) on GitHub.
+
+- Scope: GLOBAL
+- Type: String
+- Default value: `tidb-statements.log`
```
Comment (Oreoxmt) [inline system-variables.md:3819]:
```suggestion
- Default value: `"tidb-statements.log"`
```

File: system-variables.md [hunk 3]
```diff
@@ -3797,6 +3797,63 @@ For details, see [Identify Slow Queries](/identify-slow-queries.md).
...
+- This variable is read-only. It specifies the file to which persistent data is written when [statements summary persistence](/statement-summary-tables.md#persist-statements-summary) is enabled. The value of this variable is the same as that of the configuration item [`tidb_stmt_summary_enable_persistent`](/tidb-configuration-file.md#tidb_stmt_summary_enable_persistent-new-in-v660).
```
Comment (Oreoxmt) [inline system-variables.md:3820]:
```suggestion
- This variable is read-only. It specifies the file to which persistent data is written when [statements summary persistence](/statement-summary-tables.md#persist-statements-summary) is enabled. The value of this variable is the same as that of the configuration item [`tidb_stmt_summary_filename`](/tidb-configuration-file.md#tidb_stmt_summary_filename-new-in-v660).
```

File: system-variables.md [hunk 4]
```diff
@@ -3797,6 +3797,63 @@ For details, see [Identify Slow Queries](/identify-slow-queries.md).
...
+### tidb_stmt_summary_file_max_backups <span class="version-mark">New in v6.6.0</span>
+
+> **Warning:**
+>
+> Statements summary persistence is an experimental feature. It is not recommended that you use it in the production environment. This feature might be changed or removed without prior notice. If you find a bug, you can report an [issue](https://github.com/pingcap/tidb/issues) on GitHub.
+
+- Scope: GLOBAL
+- Type: Integer
+- Default value: `0`
+- This variable is read-only. It specifies the maximum number of data files that can be persisted. when [statements summary persistence](/statement-summary-tables.md#persist-statements-summary) is enabled. The value of this variable is the same as that of the configuration item [`tidb_stmt_summary_enable_persistent`](/tidb-configuration-file.md#tidb_stmt_summary_enable_persistent-new-in-v660).
```
Comment (Oreoxmt) [inline system-variables.md:3831]:
```suggestion
- This variable is read-only. It specifies the maximum number of data files that can be persisted. when [statements summary persistence](/statement-summary-tables.md#persist-statements-summary) is enabled. The value of this variable is the same as that of the configuration item [`tidb_stmt_summary_file_max_backups`](/tidb-configuration-file.md#tidb_stmt_summary_file_max_backups-new-in-v660).
```

File: system-variables.md [hunk 5]
```diff
@@ -3797,6 +3797,63 @@ For details, see [Identify Slow Queries](/identify-slow-queries.md).
...
+
+> **Warning:**
+>
+> Statements summary persistence is an experimental feature. It is not recommended that you use it in the production environment. This feature might be changed or removed without prior notice. If you find a bug, you can report an [issue](https://github.com/pingcap/tidb/issues) on GitHub.
+
+- Scope: GLOBAL
+- Type: Integer
+- Default value: `3`
+- Unit: day
+- This variable is read-only. It specifies the maximum number of days to keep persistent data files when [statements summary persistence](/statement-summary-tables.md#persist-statements-summary) is enabled. The value of this variable is the same as that of the configuration item [`tidb_stmt_summary_enable_persistent`](/tidb-configuration-file.md#tidb_stmt_summary_enable_persistent-new-in-v660).
```
Comment (Oreoxmt) [inline system-variables.md:3843]:
```suggestion
- This variable is read-only. It specifies the maximum number of days to keep persistent data files when [statements summary persistence](/statement-summary-tables.md#persist-statements-summary) is enabled. The value of this variable is the same as that of the configuration item [`tidb_stmt_summary_file_max_days`](/tidb-configuration-file.md#tidb_stmt_summary_file_max_days-new-in-v660).
```

File: system-variables.md [hunk 6]
```diff
@@ -3797,6 +3797,63 @@ For details, see [Identify Slow Queries](/identify-slow-queries.md).
...
+
+> **Warning:**
+>
+> Statements summary persistence is an experimental feature. It is not recommended that you use it in the production environment. This feature might be changed or removed without prior notice. If you find a bug, you can report an [issue](https://github.com/pingcap/tidb/issues) on GitHub.
+
+- Scope: GLOBAL
+- Type: Integer
+- Default value: `64`
+- Unit: MiB
+- This variable is read-only. It specifies the maximum size of a persistent data file when [statements summary persistence](/statement-summary-tables.md#persist-statements-summary) is enabled. The value of this variable is the same as that of the configuration item [`tidb_stmt_summary_enable_persistent`](/tidb-configuration-file.md#tidb_stmt_summary_enable_persistent-new-in-v660).
```
Comment (Oreoxmt) [inline system-variables.md:3855]:
```suggestion
- This variable is read-only. It specifies the maximum size of a persistent data file when [statements summary persistence](/statement-summary-tables.md#persist-statements-summary) is enabled. The value of this variable is the same as that of the configuration item [`tidb_stmt_summary_file_max_size`](/tidb-configuration-file.md#tidb_stmt_summary_file_max_size-new-in-v660).
```

**Review by Oreoxmt — APPROVED** [review]
Rest LGTM

**Review by Oreoxmt — APPROVED** [review]

**Review by mornyx — APPROVED** [review]

File: system-variables.md [hunk 7]
```diff
@@ -3797,6 +3797,67 @@ For details, see [Identify Slow Queries](/identify-slow-queries.md).
 - Unit: Milliseconds
 - This variable controls whether to enable the synchronously loading statistics feature. The value `0` means that the feature is disabled. To enable the feature, you can set this variable to a timeout (in milliseconds) that SQL optimization can wait for at most to synchronously load complete column statistics. For details, see [Load statistics](/statistics.md#load-statistics).
 
+<CustomContent platform="tidb">
+
```
Comment (qiancai) [inline system-variables.md:3801]:
```suggestion
```

File: system-variables.md [hunk 8]
```diff
@@ -3797,6 +3797,67 @@ For details, see [Identify Slow Queries](/identify-slow-queries.md).
...
+### tidb_stmt_summary_enable_persistent <span class="version-mark">New in v6.6.0</span>
+
```
Comment (qiancai) [inline system-variables.md:3803]:
```suggestion

<CustomContent platform="tidb-cloud">

> **Note:**
>
> This TiDB variable is not applicable to TiDB Cloud.

</CustomContent>

```

File: system-variables.md [hunk 9]
```diff
@@ -3797,6 +3797,67 @@ For details, see [Identify Slow Queries](/identify-slow-queries.md).
...
+> **Warning:**
+>
+> Statements summary persistence is an experimental feature. It is not recommended that you use it in the production environment. This feature might be changed or removed without prior notice. If you find a bug, you can report an [issue](https://github.com/pingcap/tidb/issues) on GitHub.
+
+- Scope: GLOBAL
+- Type: Boolean
+- Default value: `OFF`
+- This variable is read-only. It controls whether to enable [statements summary persistence](/statement-summary-tables.md#persist-statements-summary). The value of this variable is the same as that of the configuration item [`tidb_stmt_summary_enable_persistent`](/tidb-configuration-file.md#tidb_stmt_summary_enable_persistent-new-in-v660).
```
Comment (qiancai) [inline system-variables.md:3811]:
```suggestion
- This variable is read-only. It controls whether to enable [statements summary persistence](/statement-summary-tables.md#persist-statements-summary). 

<CustomContent platform="tidb">

- The value of this variable is the same as that of the configuration item [`tidb_stmt_summary_enable_persistent`](/tidb-configuration-file.md#tidb_stmt_summary_enable_persistent-new-in-v660).

</CustomContent>
```

File: system-variables.md [hunk 10]
```diff
@@ -3797,6 +3797,67 @@ For details, see [Identify Slow Queries](/identify-slow-queries.md).
...
+
+### tidb_stmt_summary_filename <span class="version-mark">New in v6.6.0</span>
```
Comment (shichun-0415) [inline system-variables.md:3813]:
```suggestion
<CustomContent platform="tidb">

### tidb_stmt_summary_filename <span class="version-mark">New in v6.6.0</span>

<CustomContent platform="tidb-cloud">
	
> **Note:**
>
> This TiDB variable is not applicable to TiDB Cloud.
	
</CustomContent>
```

File: system-variables.md [hunk 11]
```diff
@@ -3797,6 +3797,67 @@ For details, see [Identify Slow Queries](/identify-slow-queries.md).
...
+
+> **Warning:**
+>
+> Statements summary persistence is an experimental feature. It is not recommended that you use it in the production environment. This feature might be changed or removed without prior notice. If you find a bug, you can report an [issue](https://github.com/pingcap/tidb/issues) on GitHub.
+
+- Scope: GLOBAL
+- Type: String
+- Default value: `"tidb-statements.log"`
+- This variable is read-only. It specifies the file to which persistent data is written when [statements summary persistence](/statement-summary-tables.md#persist-statements-summary) is enabled. The value of this variable is the same as that of the configuration item [`tidb_stmt_summary_filename`](/tidb-configuration-file.md#tidb_stmt_summary_filename-new-in-v660).
```
Comment (shichun-0415) [inline system-variables.md:3822]:
```suggestion
- This variable is read-only. It specifies the file to which persistent data is written when [statements summary persistence](/statement-summary-tables.md#persist-statements-summary) is enabled. 

<CustomContent platform="tidb">

- The value of this variable is the same as that of the configuration item [`tidb_stmt_summary_filename`](/tidb-configuration-file.md#tidb_stmt_summary_filename-new-in-v660).

</CustomContent>
```

File: system-variables.md [hunk 12]
```diff
@@ -3797,6 +3797,67 @@ For details, see [Identify Slow Queries](/identify-slow-queries.md).
...
+
+### tidb_stmt_summary_file_max_backups <span class="version-mark">New in v6.6.0</span>
```
Comment (shichun-0415) [inline system-variables.md:3824]:
```suggestion
<CustomContent platform="tidb">

### tidb_stmt_summary_file_max_backups <span class="version-mark">New in v6.6.0</span>
<CustomContent platform="tidb-cloud">
	
> **Note:**
>
> This TiDB variable is not applicable to TiDB Cloud.
	
</CustomContent>
```

File: system-variables.md [hunk 13]
```diff
@@ -3797,6 +3797,67 @@ For details, see [Identify Slow Queries](/identify-slow-queries.md).
...
+
+> **Warning:**
+>
+> Statements summary persistence is an experimental feature. It is not recommended that you use it in the production environment. This feature might be changed or removed without prior notice. If you find a bug, you can report an [issue](https://github.com/pingcap/tidb/issues) on GitHub.
+
+- Scope: GLOBAL
+- Type: Integer
+- Default value: `0`
+- This variable is read-only. It specifies the maximum number of data files that can be persisted when [statements summary persistence](/statement-summary-tables.md#persist-statements-summary) is enabled. The value of this variable is the same as that of the configuration item [`tidb_stmt_summary_file_max_backups`](/tidb-configuration-file.md#tidb_stmt_summary_file_max_backups-new-in-v660).
```
Comment (shichun-0415) [inline system-variables.md:3833]:
```suggestion
- This variable is read-only. It specifies the maximum number of data files that can be persisted when [statements summary persistence](/statement-summary-tables.md#persist-statements-summary) is enabled.

<CustomContent platform="tidb">

- The value of this variable is the same as that of the configuration item [`tidb_stmt_summary_file_max_backups`](/tidb-configuration-file.md#tidb_stmt_summary_file_max_backups-new-in-v660).

</CustomContent>
```

File: system-variables.md [hunk 14]
```diff
@@ -3797,6 +3797,67 @@ For details, see [Identify Slow Queries](/identify-slow-queries.md).
...
+
+### tidb_stmt_summary_file_max_days <span class="version-mark">New in v6.6.0</span>
```
Comment (shichun-0415) [inline system-variables.md:3835]:
```suggestion
<CustomContent platform="tidb">

### tidb_stmt_summary_file_max_days <span class="version-mark">New in v6.6.0</span>

> **Note:**
>
> This TiDB variable is not applicable to TiDB Cloud.
	
</CustomContent>
```

File: system-variables.md [hunk 15]
```diff
@@ -3797,6 +3797,67 @@ For details, see [Identify Slow Queries](/identify-slow-queries.md).
...
+
+> **Warning:**
+>
+> Statements summary persistence is an experimental feature. It is not recommended that you use it in the production environment. This feature might be changed or removed without prior notice. If you find a bug, you can report an [issue](https://github.com/pingcap/tidb/issues) on GitHub.
+
+- Scope: GLOBAL
+- Type: Integer
+- Default value: `3`
+- Unit: day
+- This variable is read-only. It specifies the maximum number of days to keep persistent data files when [statements summary persistence](/statement-summary-tables.md#persist-statements-summary) is enabled. The value of this variable is the same as that of the configuration item [`tidb_stmt_summary_file_max_days`](/tidb-configuration-file.md#tidb_stmt_summary_file_max_days-new-in-v660).
```
Comment (shichun-0415) [inline system-variables.md:3845]:
```suggestion
- This variable is read-only. It specifies the maximum number of days to keep persistent data files when [statements summary persistence](/statement-summary-tables.md#persist-statements-summary) is enabled.

<CustomContent platform="tidb">

The value of this variable is the same as that of the configuration item [`tidb_stmt_summary_file_max_days`](/tidb-configuration-file.md#tidb_stmt_summary_file_max_days-new-in-v660).

</CustomContent>
```

File: system-variables.md [hunk 16]
```diff
@@ -3797,6 +3797,67 @@ For details, see [Identify Slow Queries](/identify-slow-queries.md).
...
+
+### tidb_stmt_summary_file_max_size <span class="version-mark">New in v6.6.0</span>
```
Comment (shichun-0415) [inline system-variables.md:3847]:
```suggestion
<CustomContent platform="tidb">

### tidb_stmt_summary_file_max_size <span class="version-mark">New in v6.6.0</span>

> **Note:**
>
> This TiDB variable is not applicable to TiDB Cloud.
	
</CustomContent>
```

File: system-variables.md [hunk 17]
```diff
@@ -3797,6 +3797,67 @@ For details, see [Identify Slow Queries](/identify-slow-queries.md).
...
+
+> **Warning:**
+>
+> Statements summary persistence is an experimental feature. It is not recommended that you use it in the production environment. This feature might be changed or removed without prior notice. If you find a bug, you can report an [issue](https://github.com/pingcap/tidb/issues) on GitHub.
+
+- Scope: GLOBAL
+- Type: Integer
+- Default value: `64`
+- Unit: MiB
+- This variable is read-only. It specifies the maximum size of a persistent data file when [statements summary persistence](/statement-summary-tables.md#persist-statements-summary) is enabled. The value of this variable is the same as that of the configuration item [`tidb_stmt_summary_file_max_size`](/tidb-configuration-file.md#tidb_stmt_summary_file_max_size-new-in-v660).
```
Comment (shichun-0415) [inline system-variables.md:3857]:
```suggestion
- This variable is read-only. It specifies the maximum size of a persistent data file when [statements summary persistence](/statement-summary-tables.md#persist-statements-summary) is enabled.

<CustomContent platform="tidb">

- The value of this variable is the same as that of the configuration item [`tidb_stmt_summary_file_max_size`](/tidb-configuration-file.md#tidb_stmt_summary_file_max_size-new-in-v660).
```

**Pull request merged**